tools/torch_export_to_onnx.py

```diff
@@ -182,6 +182,13 @@ def main():
     parser.add_argument("out_onnx_filename", type=str, help="Filename of the final ONNX model.")
     parser.add_argument("--verbosity", default=4, type=int, help="5 for all seqs (default: 4)")
     parser.add_argument("--device", type=str, default="cpu", help="'cpu' (default) or 'gpu'.")
+    parser.add_argument(
+        "--dyn_dim_min_sizes", type=dict, default=None, help="Specify min sizes for dim tags with dynamic sizes"
```

Comment:
Did you test this? What exactly did you pass as argument? I wonder how it would correctly convert it into a `dict`, and specifically with the right `Dim` objects as keys?

Reply:
I tested it by setting the `Dim` object like this
```
time_data_dim = SpatialDim('time:data')
extern_data_dict = {'data': {
        'dim_tags': [
            batch_dim,
            time_data_dim,
            FeatureDim('feat', 1),
        ],
        'dtype': 'float32',
    }}
dyn_dim_max_sizes={time_data_dim:100}
```

Reply:
I mean, how did you start this script? How did you pass `--dyn_dim_min_sizes` when starting the script?